authentication/views.py

```diff
@@ -32,7 +32,7 @@ def _keycloak_logout_url(self, user):
             user, provider=OlOpenIdConnectAuth.name
         ).first()
         id_token = user_social_auth_record.extra_data.get("id_token")
-        return f"{settings.KEYCLOAK_BASE_URL}/realms/{settings.KEYCLOAK_REALM_NAME}/protocol/openid-connect/logout?id_token_hint={id_token}"  # noqa: E501
+        return f"{settings.KEYCLOAK_BASE_URL}/realms/{settings.KEYCLOAK_REALM_NAME}/protocol/openid-connect/logout?id_token_hint={id_token}&post_logout_redirect_uri={settings.LOGOUT_REDIRECT_URL}"  # noqa: E501
```

Comment:
This isn't being urlencoded and we can fix that _and_ get rid of the line-to-long error/pragma (`E501`) by splitting this up and using `urlib.parse.urlencode()`, this also ensures it undergoes proper urlencoding:
```suggestion
        qs = urlencode({
            "id_token_hint": id_token,
            "post_logout_redirect_uri": settings.LOGOUT_REDIRECT_URL,
        })
        return f"{settings.KEYCLOAK_BASE_URL}/realms/{settings.KEYCLOAK_REALM_NAME}/protocol/openid-connect/logout?{qs}"
```

I can test this locally myself because I have keycloak setup there.